lib/lrug_helpers.rb

```diff
@@ -215,6 +215,41 @@ def has_host?(page)
     content_part_exists?('hosted_by', page) || page.data.has_key?('hosted_by')
   end
 
+  def events_calender(site_url:)
+    calendar = Icalendar::Calendar.new
+     calendar.timezone do |timezone|
+      timezone.tzid = 'Europe/London'
+    end
+    zone = ActiveSupport::TimeZone['Europe/London']
+
+    meeting_pages.take(12).each do |page|
+      url = URI.join(site_url, page.url)
+      date = page.metadata[:page][:meeting_date]
+      title = page.metadata[:page][:title]
+      hosts  = page.metadata[:page][:hosted_by]
+
+      calendar.event do |event|
+        event.uid = "lrug-monthly-#{date.strftime('%Y-%m')}"
+        event.dtstart = date.in_time_zone(zone).change(hour:18)
+        event.dtend   = date.in_time_zone(zone).change(hour:20)
```

Comment:
[question]

Forgive me if this is a silly question, but I thought that `in_time_zone` was a Rails method. How did you manage to get it to work here?